README.md

```diff
@@ -34,12 +34,15 @@ dicomsort [options...] sourceDir targetDir/<patterns>
     [-f,--forceDelete] - remove source without confirmation
     [-k,--keepGoing] - report but ignore dupicate target files
     [-v,--verbose] - print diagnostics while processing
+    [-s,--symlink] - create a symlink to dicom files in sourceDir instead of copying them
     [-t,--test] - run the built in self test (requires internet)
     [-u,--unsafe] - do not replace unsafe characters with '_' in the path
     [--help] - print this message
 
- <patterns...> is a string defining the output file and directory
-names based on the dicom tags in the file.
+ where sourceDir is directory to be scanned or "" (null string) to read file list from stdin
```

Comment:
I think it would be more standard to just omit the argument if you want it to be optional.  Then instead of checking for "" you would check against None, which would be the default value if the argument is not specified.

However in this case since leaving off the argument by mistake would just leave the program hanging waiting for data on stdin, I think it would be better to introduce and explicit flag like --fileList which would take a filename for the list of files to process or a "-" character to indicate that the list should come from stdin.

Reply:
I agree with you! I thought through those admittedly preferred options. But, I was looking for something that wouldn't impact the code too much to review easily.

With "-" passed as the sourceDir, one would have to differentiate it from other "-x" and "--argStr" style args.

With "-l" "--fileList" style, there would be the first arg to take a parameter and it would change the expectation of how many positional args followed.

I was hoping the "" solution would fit with the current code since there are so few lines of code and that it if were ripped out later, it wouldn't be a big loss.